### ISSUE TITLE
Converter options

### ISSUE THREAD

**Issue by rosell-dk** [rosell-dk/webp-convert#18]
Right now you set converter options by defining constants, like `WEBPCONVERT_EWW_KEY` and `WEBPCONVERT_GD_PNG`.

It seems a bit hackish.

With an API change coming up, lets find a better solution.

For example,

```
WebPConvert::setConverterOptions(array(
  'ewww' => array(
    'key' => 'xxx897aoefu'
  ),
  'gd' => array(
    'convert_pngs' => true
  )
));
```

**Comment by rosell-dk** [comment]
@S1SYPHOS

**Comment by S1SYPHOS** [comment]
Well, first thing that comes to mind is that most of these options (as of now) won't be needed that often - since `WebPConvert` is kinda out-of-the-box (except for quality I guess, which is passed with the `convert()` call, so that doesn't count)!

**Comment by rosell-dk** [comment]
True.
We could discard the options altogether, and then add the functionality above later. That way we will not break backwards compatibility when we introduce options (which will force us to increment major release number). Of course, ewww will not be usable, if you cannot set options. But it has problems already, right?

But it should be no big deal to program this. I can do that after this PR.

**Comment by rosell-dk** [comment]
It seems you are very eager to release this as 1.0.0 very soon ?

**Comment by S1SYPHOS** [comment]
I'm pretty excited about it, officially contributing to this project which I'm grateful for, true. But only if everything is in place and working!

**Comment by rosell-dk** [comment]
I implemented the above.
But the syntax seems a bit cumbersome, given that you often only want to set a single value.
I'm changing syntax to:

WebPConvert::setConverterOption('ewww', 'key', 'YOUR-KEY-HERE');

**Comment by rosell-dk** [comment]
Fixed in commit 5451228e2bea606b90402befc1386734265c9c9b

**Comment by rosell-dk** [comment]
README doesn't completely reflect the changes
